### ISSUE TITLE
Artifact download missing .tar extension

### ISSUE THREAD

**Issue by gregbacchus** [wercker/wercker#115]
When downloading artifacts, something has changed recently and the file extension ".tar" is no longer added to the downloaded file. This means I always need to add it manually before opening.

I am downloading by clicking on the "Download aritfact" button in the build step. Have tested in both Chrome and Safari. I get a file called "output" (no extension)

Further information: This is only the case for recent builds (not sure the exact date but in the last week). For older builds (even of the same project) it still downloads a file called "artifacts.tar". I get the same behavior across different projects.

**Comment by mwhooker** [comment]
Hi @gregbacchus 

thanks very much for the report. Looks like this behavior changed recently, and we forgot to update the artifact URL. Sending a path up now.

**Comment by gregbacchus** [comment]
Would be great to be able to set the artifact file name somehow. So that we could stamp in the name and build number or repo hash.

**Comment by mwhooker** [comment]
thanks for your feedback. if you open a new ticket for that we'll make sure it gets tracked, since this ticket will be closed once the patch is merged.

**Comment by gregbacchus** [comment]
done. thanks.
